src/halmos/__main__.py

```diff
@@ -433,12 +456,14 @@ def run(
             elif args.print_potential_counterexample:
                 print(color_warn(f'Counterexample (potentially invalid): {str_model(model, args)}'))
             else:
-                print(color_warn(f'Counterexample: unknown'))
-        else:
-            print(color_warn(f'Counterexample: unknown'))
+                print(color_warn(f'Counterexample: {result}'))
```

Comment:
at this point, `result` will be always `sat`, but we want to print a different message, because the generated counterexample is potentially invalid. this is because it includes an interpretation of `evm_div` (uninterpreted abstraction of bitvector division); see `is_valid_model()`.

so i suggest something like "counterexample is unknown", or "no valid counterexample", or more detailed explanation.

Reply:
Yes, in fact I think we should do something like slither:

```
C.destination (test.sol#3) is never initialized. It is used in:
	- f (test.sol#5-7)
Reference: https://github.com/trailofbits/slither/wiki/Vulnerabilities-Description#uninitialized-state-variables
```

or errorprone:

```
$ bazel build :hello
ERROR: example/myproject/BUILD:29:1: Java compilation in rule '//example/myproject:hello'
ShortSet.java:6: error: [CollectionIncompatibleType] Argument 'i - 1' should not be passed to this method;
its type int is not compatible with its collection's type argument Short
      s.remove(i - 1);
              ^
    (see http://errorprone.info/bugpattern/CollectionIncompatibleType)
1 error
```
 
👉 display a short but readable error message, with a machine-readable error-code (`uninitialized-state-variables`, `CollectionIncompatibleType`) that can be used to link to a longer explanation